Rd 872 components fix plugins update (#1682)
cloudify-cosmo/docs.getcloudify.org: content/cli/orch_cli/plugins.md

                              specified, the current tenant will be used
*  `--include-logs / --no-logs`  - Include logs in returned events [default: True]
*  `--json-output`              -  Output events in a consumable JSON format

> *  `-f, --force`                 Force running the update also in case a

Reviewer: ```suggestion
*  `-f, --force`                 Force running the update when the previous update has failed to finish successfully. This flag applies to both direct deployment update and cascading update to components.
```

>                               blueprint (for which the update is executed) is

Reviewer: ```suggestion

```

>                               used as a component

Reviewer: ```suggestion

```

Author: @ofercloudify this part of a message

> Force running the update when the previous update has failed to finish successfully.

Is misleading.  This is not the functionality `--force` is used for (but `--reevaluate-active-statuses` is).

Author: The `--reevaluate-active-statuses` is documented along with other changes in this PR: https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/1744
